src/Fixtures/en/frisbeeDrills.en.js

```diff
@@ -6573,6 +6577,104 @@ const fish = {
   ],
 };
 
+const trident = {
+  id: 1050,
+  type: DrillTypes.FRISBEE,
+  author: 'Monkey',
+  title: 'The Trident',
+  image: 'https://zupimages.net/up/20/47/6ksu.jpg',
+  description: 'Players practice quickly identifying a target to pass to as quickly as possible.',
+  minimalPlayersNumber: 5,
+  inGame: 'Identifying the best passing option is paramount in a game.',
+  equipmentLabel: EquipmentLabels.BASIC,
+  equipment: '2 discs for 5 players',
+  durationInMinutes: 10,
+  intensity: Intensities.MODERATE,
+  goals: [FrisbeeGoals.MARK, FrisbeeGoals.THROWING],
+  seasonTiming: SeasonTimings.ANYTIME,
+  level: Levels.BEGINNER,
+  steps: [
+    {
+      id: 1,
+      title: 'The Trident',
+      animation: tridentAnimation,
+      repetition: '',
+      rest: '',
+      instruction:
+        'The thrower is looking to play with his teammate. He is marked (forehand or backhand) by the defender. He tries to make as many passes as possible in 1 minute.\nGoal for beginner: To play on the easiest to reach.\nExperienced goal: Try to break systematically.',
+    },
+  ],
+};
+
+const firstWall = {
```

Comment:
I think it's almost the same drill as one we already have : Swinging with the cup

Maybe it should be a variant of this drill (a bit more intense because the play never stops)